rtrlib/rtr/packets.c

```diff
@@ -731,8 +736,9 @@ static int rtr_store_prefix_pdu(struct rtr_socket *rtr_socket, const void *pdu,
 static int rtr_store_router_key_pdu(struct rtr_socket *rtr_socket, const void *pdu, const unsigned int pdu_size, void **ary,
                                     unsigned int *ind, unsigned int *size)
 {
-    const enum pdu_type pdu_type = rtr_get_pdu_type(pdu);
-    assert(pdu_type == ROUTER_KEY);
+    const enum pdu_type type = rtr_get_pdu_type(pdu);
+    (void) type;
+    assert(type == ROUTER_KEY);
```

Comment:
all other functions require the ```type``` variable, hence it makes no sense to use shorter version there, see for instance [here](https://github.com/rtrlib/rtrlib/blob/master/rtrlib/rtr/packets.c#L809)